FinniversKit/Sources/Components/SelectionBoxes/Selectionbox.swift

```diff
@@ -78,7 +78,10 @@ public class Selectionbox: UIView {
 
     public var title: String? {
         get { return titleLabel.text }
-        set { titleLabel.text = newValue }
+        set {
+            titleLabel.text = newValue
+            setupSubviews()
```

Comment:
Not necessary to do a new setup each time the title is set.

Reply:
We need to hide the title if the label text is set to nil

Reply:
Yes but the label can just toggle isHidden based on presence of the text.